### PR TITLE
Fix overrides for test files

### PR DESCRIPTION
**Pull request by RusinovAnton**
add scoped `yola/` prefix for override rule

### CODE REVIEW

**Comment by RusinovAnton** [comment]
@yola/frontend-engineering please review

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-plugin-yola",
-  "version": "0.2.1",
+  "version": "0.2.1-dev.1",
```
Comment (AnastasiiaLisovska) [inline package.json]: shouldnt here be <next_version>-dev.1?

Comment (RusinovAnton) [reply package.json]: @AnastasiiaLisovska our flow of version bump is that we're open separate PR with new version proposal after changes been approved. So here I assume that there is no new version yet while my changes get approve or discard.

File: src/get-config-base.js [hunk 2]
```diff
@@ -24,11 +24,12 @@ module.exports = () => ({
       ],
       env: {
         mocha: true,
+        jest: true,
       },
       rules: {
         // Override `import/no-extraneous-dependencies` rule for test files
```
Comment (AnastasiiaLisovska) [inline src/get-config-base.js]: I think comment should also be updated with `yola/`

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,12 +1,16 @@
 # Change log
 
+## dev
+
+* Fix overrides for test files
```
Comment (AnastasiiaLisovska) [inline CHANGELOG.md]: what about changing it to `add scoped yola/ prefix for override rule`? I think its more informative

Comment (RusinovAnton) [reply CHANGELOG.md]: Changelog must have description of things that is important to package user. The fix itself is not so important, though anyone who is interested can come and see commit of what exactly was broken.

**Comment by AnastasiiaLisovska** [comment]
LGTM :+1:

**Pull request merged**